plugins/techdocs/src/reader/components/TechDocsReaderPage/TechDocsReaderPage.tsx

```diff
@@ -143,6 +147,27 @@ export const TechDocsReaderLayout = (props: TechDocsReaderLayoutProps) => {
   );
 };
 
+function TechDocsAuthProvider({ children }: { children: ReactNode }) {
+  const app = useApp();
+  const { Progress } = app.getComponents();
+
+  const techdocsApi = useApi(techdocsApiRef);
+
+  const { loading, error } = useAsync(async () => {
+    return await techdocsApi.issueUserCookie();
```

Comment:
Is it possible to detect if protected by default is enabled on the frontend?
I'm asking because if it's disabled, I don't want to call the cookie endpoint.
Wouldn't it be better if the backend handled it?

Reply:
I haven't checked the code yet, but the intent isn't that there should be a 401. 🤔

Reply:
Fixed with this suggestion 🙏🏻 :
https://github.com/backstage/backstage/pull/23437#discussion_r1519644853